src/CarbonAware.CLI/src/Commands/Emissions/EmissionForecastCommand.cs

```diff
@@ -0,0 +1,100 @@
+﻿using CarbonAware.Aggregators.CarbonAware;
+using CarbonAware.CLI.Common;
+using CarbonAware.CLI.Model;
+using CarbonAware.Tools.WattTimeClient.Model;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Text.Json;
+
+namespace CarbonAware.CLI.Commands.Emissions;
+
+class EmissionsForecastCommand : Command
+{
+    private readonly Option<string[]> _requiredLocation = CommonOptions.RequiredLocationOption;
+    
+    private readonly Option<DateTimeOffset?> _dataEndTime = new Option<DateTimeOffset?>(
+                new string[] { "--data-end-time", "-e" },
+            LocalizableStrings.DataEndAtDecsription)
+    {
+        Arity = ArgumentArity.ZeroOrOne,
+    };
+    private readonly Option<DateTimeOffset?> _dataRequestedAt = new Option<DateTimeOffset?>(
+                new string[] { "--data-requested-at", "-r" },
```

Comment:
In addition to my larger comment above, I feel that this should change because it confuses the semantics.  The `data` prefix is meant to signal the relationship between the parameter and the `forecastData` field.  `requested-at` is a parameter for the forecast object itself, like location, and thus does not warrant a prefix.